src/libraries/System.IO.Compression/tests/System.IO.Compression.Tests.csproj

```diff
@@ -1,5 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <EnableLibraryImportGenerator>true</EnableLibraryImportGenerator>
```

Comment:
Why is this necessary?

Reply:
This was necessary before I switched to the new API: it made the interop methods work. It is probably no longer necessary. I will remove it once I get my Codespaces instance to behave and test the new version as I can't compile it right now.